### PR TITLE
47 documentation on monitoring concept for frontend-e8

### PR DESCRIPTION
**Pull request by Koufan-De-King**

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -2,7 +2,50 @@ import React from "react";
...
+import {
+  getWebInstrumentations,
+  initializeFaro,
+  ReactIntegration,
+  ReactRouterVersion,
+} from "@grafana/faro-react";
+import { TracingInstrumentation } from "@grafana/faro-web-tracing";
 
+initializeFaro({
+  // Replace with ADORSYS' grafana cloud webank-userapp collector url
```
Comment (NkwaTambe) [inline src/main.tsx:21]: please look into these comment..

**Review by NkwaTambe — CHANGES_REQUESTED** [review]
This is just a documentation.. so why add code ?????? please review again

File: Docs/Production/GrafanaMonitoring.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+# Grafana Faro Integration Guide
+
+This guide will help you set up monitoring and logging for your systems using Grafana Faro. This documentation is divided into four main sections: 
+1. [**SET UP**](#1-setup)
+
+2. [**ARCHITECTURE OF LOGGING AND MONITORING**](#2-architecture-of-logging-and-monitoring) 
+
+3. [**FURTHER CUSTOMIZATIONS**](#3-further-customizations)
+
+4. [**PACKAGES USED**](#4-packages-used)
```
Comment (NkwaTambe) [inline Docs/Production/GrafanaMonitoring.md:11]: links not opening

**Review by Elwizzy12 — APPROVED** [review]

**Review by Elwizzy12 — APPROVED** [review]
good!

**Review by Arielpetit — APPROVED** [review]

**Review by NkwaTambe — APPROVED** [review]

**Pull request merged**